### ISSUE TITLE
money should not be stored as number

### ISSUE THREAD

**Issue by growandwin** [flash-oss/medici#29]
Hi, nice library, but money should not be stored in floats - this is just very risky and a great pitfall. 

Maybe you could implement dinero? 

https://v2.dinerojs.com/docs

**Comment by koresar** [comment]
Hello.
Very much agree about calculating numbers. But I can't see why you propose calculation library to store numbers. I couldn't find by your link how the dinero numbers should be stored.
Moreover, MongoDB stores JS Numbers natively. Whereas dinero data is not native for MongoDB.

Dinero is not a storage library, but a calculation library. Proposing to "store dinero objects in MongoDB" sounds like a significant performance degradation for medici.

Also, MongoDB has calculations built in. When medici calculates balances it uses $sum pipeline operator. I'm not sure if it uses some safety algorithms.

In other words.
If you want medici to calculate things differently you should start from MongoDB built-in $sum pipeline operator.

But we are happy to accept any Pull Requests you are willing to contribute.
